pillar/edx/ansible_vars/residential.sls

```diff
@@ -174,7 +174,11 @@ edx:
         RESTRICT_ENROLL_SOCIAL_PROVIDERS:
           - mit-kerberos
         ENABLE_THIRD_PARTY_ONLY_AUTH: True
+        {% if 'koa' not in grains.get('edx_codename') %}
```

Comment:
We don't need a conditional here either.

Reply:
Why not?

Reply:
I guess it doesn't matter either way.